nservicebus/throughput-tool/index.md

```diff
@@ -1,16 +1,26 @@
 ---
-title: Endpoint Throughput tool
+title: Collecting usage data using legacy Endpoint Throughput tool
 summary: Use the Particular endpoint throughput counter tool to measure the usage of an NServiceBus system.
 reviewed: 2024-05-22
 related:
   - servicepulse/usage
+redirects:
+  - nservicebus/throughput-tool/azure-service-bus
+  - nservicebus/throughput-tool/amazon-sqs
+  - nservicebus/throughput-tool/rabbitmq
+  - nservicebus/throughput-tool/sql-transport
+  - nservicebus/throughput-tool/postgresql-transport
+  - nservicebus/throughput-tool/service-control
+  - nservicebus/throughput-tool/determine-transport
+  - nservicebus/throughput-tool/faq
 ---
 
-> [!NOTE]
-> The recommended method of measuring the throughput of an NServiceBus system is via [ServicePulse](./../../servicepulse/usage.md).
-> The endpoint throughput counter tool is offered as an alternate option for customers that aren't able to use ServicePulse.
+> [!WARNING]
+> The recommended method of measuring the throughput of an NServiceBus system is via [ServicePulse](./../../servicepulse/usage.md). Once configured, ServicePulse provides the ability to report throughput instantly at any time, without needing to run an external tool or wait for data collection.
+>
+> The legacy endpoint throughput counter tool is offered as an alternate option for customers that aren't able to use ServicePulse.
```

Comment:
We had a stronger statement on the old landing page. Would it make sense to bring that here?

```suggestion
> [!WARNING]
> Starting 1 January 2026, it will become mandatory for all projects in active development to use [ServicePulse](./../../servicepulse/usage.md) for usage data collection. Once configured, ServicePulse provides the ability to report throughput instantly at any time, without needing to run an external tool or wait for data collection.
>
> The legacy endpoint throughput counter tool is offered as an alternate option for customers that aren't able to use ServicePulse. If you have any issues installing ServicePulse in your environment, please reach out so we can help find a path forward.
```